config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -643,7 +643,7 @@ private ApplicationContext getContext() {
 	 *
 	 * @see #requestMatcher(RequestMatcher)
 	 *
-	 * @return
+	 * @return the {@link ExpressionUrlAuthorizationConfigurer}#ExpressionInterceptUrlRegistry for further customizations
```

Comment:
In my experience, the '#' refers to methods, not subclasses. "{@link ExpressionUrlAuthorizationConfigurer.ExpressionInterceptUrlRegistry}" would probably be more consistent.

That said, based on some of your changes to UrlAuthorizationConfigurer javadocs, I think that "{@link ExpressionUrlAuthorizationConfigurer}" would be more in line with those.